tests/thread_msg/main.c

```diff
@@ -53,7 +53,7 @@ void *thread1(void *arg)
 void *thread2(void *arg)
 {
     (void) arg;
-    puts("THREAD 2\n");
+    puts("THREAD 2 start\n");
```

Comment:
Not needed: why change this, instead of keeping it simple as it was? IMHO it doesn't add any value or information, it just uses more memory.